lms/djangoapps/verify_student/views.py

```diff
@@ -1006,6 +1007,21 @@ def results_callback(request):
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
+        if checkpoints:
+            course_key = checkpoints[0].course_id
+            credit_requirement = get_credit_requirement(
+                course_key, "reverification", checkpoints[0].checkpoint_name
+            )
+            if credit_requirement is not None:
```

Comment:
`get_credit_requirement` will return empty array [] if there is no requirement so this check should be `if redit_requirement`